albumentations/core/composition.py

```diff
@@ -210,12 +263,15 @@ def _check_data_post_transform(self, data):
         return data
 
     def _to_dict(self):
+        self.is_check_args = True
```

Comment:
Oh my fault. We don't need this var there, because it is not an argument in `__init__`. Sorry.